[MinGW][Windows] Error compiling

I'm compiling SVT-AV1 for Windows under linux, using a cross compiler (see ffmpeg-windows-build-helpers repository).

Basically the compiling is performed as follows:
```
cmake -G"Unix Makefiles" . -DCMAKE_SYSTEM_NAME=Windows -DCMAKE_FIND_ROOT_PATH=$install_prefix -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER -DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY -DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY -DCMAKE_RANLIB=${cross_prefix}ranlib -DCMAKE_C_COMPILER=${cross_prefix}gcc -DCMAKE_CXX_COMPILER=${cross_prefix}g++ -DCMAKE_RC_COMPILER=${cross_prefix}windres $extra_args -DCMAKE_BUILD_TYPE=Release -DCPPAN_BUILD=OFF -DENABLE_STATIC_RUNTIME=1 -DBUILD_SHARED_LIBS=1 -DBUILD_STATIC_LIBS=0
make -j 3
make install -j 3
```

The last commit that I checked was `3a0d50babd220d6f991c649824c115dc2e6ae5c3 6/4/2020`, but with the new one is not working `ba72bc8511ed6a31544152590740e85f99a41300 6/16/2020`. I don't known exactly in what commit the issue occurs. But the errors are the follow:

```
/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_blend_a64_mask_sse4_1[.refptr.aom_blend_a64_mask_sse4_1]+0x0): undefined reference to `aom_blend_a64_mask_sse4_1'
collect2: error: ld returned 1 exit status
make[2]: *** [../Bin/Release/libSvtAv1Dec.dll] Error 1
make[1]: *** [Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```
```
[ 89%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/palette.c.obj
[ 90%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/ransac.c.obj
[ 90%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/vector.c.obj
[ 90%] Built target ENCODER_CODEC
make: *** [all] Error 2
```

I had some difficulties getting the log but here  is what I get:
```
[ 74%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbResourceCoordinationResults.c.obj
[ 75%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbRestProcess.c.obj
[ 75%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbRestorationPick.c.obj
[ 75%] Building C object Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/Codec/EbDecProcessBlock.c.obj
[ 76%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbSegmentation.c.obj
[ 77%] Building C object Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/Codec/EbDecProcessFrame.c.obj
[ 77%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbSequenceControlSet.c.obj
[ 78%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbSourceBasedOperationsProcess.c.obj
[ 78%] Building C object Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/Codec/EbDecRestoration.c.obj
[ 79%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbString.c.obj
[ 79%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbTemporalFiltering.c.obj
[ 80%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbTransQuantBuffers.c.obj
[ 80%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/EbTransforms.c.obj
[ 81%] Building C object Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/Codec/EbDecSuperRes.c.obj
[ 82%] Building C object Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/Codec/EbDecUtils.c.obj
[ 82%] Linking C shared library ../../../../Bin/Release/libSvtAv1Dec.dll
[ 83%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/aom_dsp_rtcd.c.obj

/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_8x4_avx2[.refptr.eb_aom_highbd_paeth_predictor_8x4_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_8x4_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_8x32_avx2[.refptr.eb_aom_highbd_paeth_predictor_8x32_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_8x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_8x16_avx2[.refptr.eb_aom_highbd_paeth_predictor_8x16_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_8x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_64x64_avx2[.refptr.eb_aom_highbd_paeth_predictor_64x64_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_64x64_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_64x32_avx2[.refptr.eb_aom_highbd_paeth_predictor_64x32_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_64x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_64x16_avx2[.refptr.eb_aom_highbd_paeth_predictor_64x16_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_64x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_4x8_avx2[.refptr.eb_aom_highbd_paeth_predictor_4x8_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_4x8_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_4x4_avx2[.refptr.eb_aom_highbd_paeth_predictor_4x4_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_4x4_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_4x16_avx2[.refptr.eb_aom_highbd_paeth_predictor_4x16_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_4x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_32x8_avx2[.refptr.eb_aom_highbd_paeth_predictor_32x8_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_32x8_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_32x64_avx2[.refptr.eb_aom_highbd_paeth_predictor_32x64_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_32x64_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_32x32_avx2[.refptr.eb_aom_highbd_paeth_predictor_32x32_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_32x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_32x16_avx2[.refptr.eb_aom_highbd_paeth_predictor_32x16_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_32x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_2x2_avx2[.refptr.eb_aom_highbd_paeth_predictor_2x2_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_2x2_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_16x8_avx2[.refptr.eb_aom_highbd_paeth_predictor_16x8_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_16x8_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_16x64_avx2[.refptr.eb_aom_highbd_paeth_predictor_16x64_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_16x64_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_16x4_avx2[.refptr.eb_aom_highbd_paeth_predictor_16x4_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_16x4_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_16x32_avx2[.refptr.eb_aom_highbd_paeth_predictor_16x32_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_16x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_paeth_predictor_16x16_avx2[.refptr.eb_aom_highbd_paeth_predictor_16x16_avx2]+0x0): undefined reference to `eb_aom_highbd_paeth_predictor_16x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_dr_prediction_z3_avx2[.refptr.eb_av1_highbd_dr_prediction_z3_avx2]+0x0): undefined reference to `eb_av1_highbd_dr_prediction_z3_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_dr_prediction_z2_avx2[.refptr.eb_av1_highbd_dr_prediction_z2_avx2]+0x0): undefined reference to `eb_av1_highbd_dr_prediction_z2_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_dr_prediction_z1_avx2[.refptr.eb_av1_highbd_dr_prediction_z1_avx2]+0x0): undefined reference to `eb_av1_highbd_dr_prediction_z1_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_dr_prediction_z3_avx2[.refptr.eb_av1_dr_prediction_z3_avx2]+0x0): undefined reference to `eb_av1_dr_prediction_z3_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_dr_prediction_z2_avx2[.refptr.eb_av1_dr_prediction_z2_avx2]+0x0): undefined reference to `eb_av1_dr_prediction_z2_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_dr_prediction_z1_avx2[.refptr.eb_av1_dr_prediction_z1_avx2]+0x0): undefined reference to `eb_av1_dr_prediction_z1_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_64x64_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_64x64_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_64x64_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_64x32_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_64x32_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_64x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_64x16_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_64x16_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_64x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_32x64_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_32x64_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_32x64_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_32x32_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_32x32_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_32x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_32x16_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_32x16_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_32x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_32x8_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_32x8_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_32x8_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_8x8_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_8x8_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_8x8_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_8x4_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_8x4_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_8x4_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_8x32_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_8x32_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_8x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_8x16_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_8x16_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_8x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_16x8_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_16x8_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_16x8_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_16x64_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_16x64_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_16x64_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_16x4_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_16x4_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_16x4_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_16x32_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_16x32_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_16x32_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_aom_highbd_smooth_v_predictor_16x16_avx2[.refptr.eb_aom_highbd_smooth_v_predictor_16x16_avx2]+0x0): undefined reference to `eb_aom_highbd_smooth_v_predictor_16x16_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_highbd_subtract_block_sse2[.refptr.aom_highbd_subtract_block_sse2]+0x0): undefined reference to `aom_highbd_subtract_block_sse2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_highbd_blend_a64_d16_mask_avx2[.refptr.aom_highbd_blend_a64_d16_mask_avx2]+0x0): undefined reference to `aom_highbd_blend_a64_d16_mask_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_lowbd_blend_a64_d16_mask_avx2[.refptr.aom_lowbd_blend_a64_d16_mask_avx2]+0x0): undefined reference to `aom_lowbd_blend_a64_d16_mask_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_subtract_block_avx2[.refptr.aom_subtract_block_avx2]+0x0): undefined reference to `aom_subtract_block_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.av1_wedge_sse_from_residuals_avx2[.refptr.av1_wedge_sse_from_residuals_avx2]+0x0): undefined reference to `av1_wedge_sse_from_residuals_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.av1_build_compound_diffwtd_mask_highbd_avx2[.refptr.av1_build_compound_diffwtd_mask_highbd_avx2]+0x0): undefined reference to `av1_build_compound_diffwtd_mask_highbd_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.av1_build_compound_diffwtd_mask_avx2[.refptr.av1_build_compound_diffwtd_mask_avx2]+0x0): undefined reference to `av1_build_compound_diffwtd_mask_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_convolve8_vert_avx2[.refptr.aom_convolve8_vert_avx2]+0x0): undefined reference to `aom_convolve8_vert_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_convolve8_horiz_avx2[.refptr.aom_convolve8_horiz_avx2]+0x0): undefined reference to `aom_convolve8_horiz_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_jnt_convolve_y_avx2[.refptr.eb_av1_jnt_convolve_y_avx2]+0x0): undefined reference to `eb_av1_jnt_convolve_y_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_jnt_convolve_x_avx2[.refptr.eb_av1_jnt_convolve_x_avx2]+0x0): undefined reference to `eb_av1_jnt_convolve_x_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_jnt_convolve_2d_copy_avx2[.refptr.eb_av1_jnt_convolve_2d_copy_avx2]+0x0): undefined reference to `eb_av1_jnt_convolve_2d_copy_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_jnt_convolve_2d_avx2[.refptr.eb_av1_jnt_convolve_2d_avx2]+0x0): undefined reference to `eb_av1_jnt_convolve_2d_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_convolve_y_sr_avx2[.refptr.eb_av1_convolve_y_sr_avx2]+0x0): undefined reference to `eb_av1_convolve_y_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_convolve_x_sr_avx2[.refptr.eb_av1_convolve_x_sr_avx2]+0x0): undefined reference to `eb_av1_convolve_x_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_convolve_2d_copy_sr_avx2[.refptr.eb_av1_convolve_2d_copy_sr_avx2]+0x0): undefined reference to `eb_av1_convolve_2d_copy_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_convolve_2d_sr_avx2[.refptr.eb_av1_convolve_2d_sr_avx2]+0x0): undefined reference to `eb_av1_convolve_2d_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_convolve_x_sr_avx2[.refptr.eb_av1_highbd_convolve_x_sr_avx2]+0x0): undefined reference to `eb_av1_highbd_convolve_x_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_jnt_convolve_y_avx2[.refptr.eb_av1_highbd_jnt_convolve_y_avx2]+0x0): undefined reference to `eb_av1_highbd_jnt_convolve_y_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_jnt_convolve_x_avx2[.refptr.eb_av1_highbd_jnt_convolve_x_avx2]+0x0): undefined reference to `eb_av1_highbd_jnt_convolve_x_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_jnt_convolve_2d_avx2[.refptr.eb_av1_highbd_jnt_convolve_2d_avx2]+0x0): undefined reference to `eb_av1_highbd_jnt_convolve_2d_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_convolve_2d_sr_avx2[.refptr.eb_av1_highbd_convolve_2d_sr_avx2]+0x0): undefined reference to `eb_av1_highbd_convolve_2d_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_convolve_y_sr_avx2[.refptr.eb_av1_highbd_convolve_y_sr_avx2]+0x0): undefined reference to `eb_av1_highbd_convolve_y_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_jnt_convolve_2d_copy_avx2[.refptr.eb_av1_highbd_jnt_convolve_2d_copy_avx2]+0x0): undefined reference to `eb_av1_highbd_jnt_convolve_2d_copy_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_convolve_2d_copy_sr_avx2[.refptr.eb_av1_highbd_convolve_2d_copy_sr_avx2]+0x0): undefined reference to `eb_av1_highbd_convolve_2d_copy_sr_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_wiener_convolve_add_src_avx2[.refptr.eb_av1_wiener_convolve_add_src_avx2]+0x0): undefined reference to `eb_av1_wiener_convolve_add_src_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.avc_style_luma_interpolation_filter_helper_ssse3[.refptr.avc_style_luma_interpolation_filter_helper_ssse3]+0x0): undefined reference to `avc_style_luma_interpolation_filter_helper_ssse3'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_enc_msb_pack2d_sse2_intrin[.refptr.eb_enc_msb_pack2d_sse2_intrin]+0x0): undefined reference to `eb_enc_msb_pack2d_sse2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.convert_16bit_to_8bit_avx2[.refptr.convert_16bit_to_8bit_avx2]+0x0): undefined reference to `convert_16bit_to_8bit_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.convert_8bit_to_16bit_avx2[.refptr.convert_8bit_to_16bit_avx2]+0x0): undefined reference to `convert_8bit_to_16bit_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.cfl_luma_subsampling_420_hbd_avx2[.refptr.cfl_luma_subsampling_420_hbd_avx2]+0x0): undefined reference to `cfl_luma_subsampling_420_hbd_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.cfl_luma_subsampling_420_lbd_avx2[.refptr.cfl_luma_subsampling_420_lbd_avx2]+0x0): undefined reference to `cfl_luma_subsampling_420_lbd_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_enc_un_pack8_bit_data_avx2_intrin[.refptr.eb_enc_un_pack8_bit_data_avx2_intrin]+0x0): undefined reference to `eb_enc_un_pack8_bit_data_avx2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.unpack_avg_safe_sub_avx2_intrin[.refptr.unpack_avg_safe_sub_avx2_intrin]+0x0): undefined reference to `unpack_avg_safe_sub_avx2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.unpack_avg_avx2_intrin[.refptr.unpack_avg_avx2_intrin]+0x0): undefined reference to `unpack_avg_avx2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.unpack_avg_sse2_intrin[.refptr.unpack_avg_sse2_intrin]+0x0): undefined reference to `unpack_avg_sse2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_inv_txfm_add_ssse3[.refptr.eb_av1_inv_txfm_add_ssse3]+0x0): undefined reference to `eb_av1_inv_txfm_add_ssse3'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.c_pack_avx2_intrin[.refptr.c_pack_avx2_intrin]+0x0): undefined reference to `c_pack_avx2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.compressed_packmsb_avx2_intrin[.refptr.compressed_packmsb_avx2_intrin]+0x0): undefined reference to `compressed_packmsb_avx2_intrin'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_inv_txfm_add_avx2[.refptr.eb_av1_inv_txfm_add_avx2]+0x0): undefined reference to `eb_av1_inv_txfm_add_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.eb_av1_highbd_inv_txfm_add_avx2[.refptr.eb_av1_highbd_inv_txfm_add_avx2]+0x0): undefined reference to `eb_av1_highbd_inv_txfm_add_avx2'
/home/user/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld: CMakeFiles/SvtAv1Dec.dir/objects.a(common_dsp_rtcd.c.obj):common_dsp_rtcd.c:(.rdata$.refptr.aom_blend_a64_mask_sse4_1[.refptr.aom_blend_a64_mask_sse4_1]+0x0): undefined reference to `aom_blend_a64_mask_sse4_1'
collect2: error: ld returned 1 exit status
make[2]: *** [../Bin/Release/libSvtAv1Dec.dll] Error 1
make[1]: *** [Source/Lib/Decoder/CMakeFiles/SvtAv1Dec.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 84%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/av1me.c.obj
[ 84%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/corner_detect.c.obj
[ 85%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/corner_match.c.obj
[ 86%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/fft.c.obj
[ 86%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/global_motion.c.obj
[ 87%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/hash.c.obj
[ 87%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/hash_motion.c.obj
[ 88%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/noise_model.c.obj
[ 89%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/noise_util.c.obj
[ 89%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/palette.c.obj
[ 90%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/ransac.c.obj
[ 90%] Building C object Source/Lib/Encoder/Codec/CMakeFiles/ENCODER_CODEC.dir/vector.c.obj
[ 90%] Built target ENCODER_CODEC
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,8 +29,19 @@ if(NOT CMAKE_SIZEOF_VOID_P EQUAL 8)
     message(WARNING "32-bit is not supported")
 endif()
 
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
-    set(X86 TRUE)
+file(WRITE ${CMAKE_BINARY_DIR}/conftest.c "
+#if defined(_M_IX86) || defined(_M_X64) || defined(__i386__) || defined(__x86_64__)
+#else
+#error \"Non-x86\"
+#endif
+int main() {}
+")
+
+try_compile(X86 ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/conftest.c)
+
+message(STATUS "x86 detected: ${X86}")
+
+if(X86)
     find_program(YASM_EXE yasm)
     option(ENABLE_NASM "Use nasm if available (Uses yasm by default if found)" OFF)
     if(YASM_EXE AND NOT CMAKE_ASM_NASM_COMPILER MATCHES "yasm" AND NOT ENABLE_NASM)
@@ -51,8 +62,6 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
         endif()
     endif()
     enable_language(ASM_NASM)
-else()
-    set(X86 FALSE)
 endif()
 
 include(GNUInstallDirs)

